examples/templates/index.html

```diff
@@ -8,3 +8,4 @@
 #}
 Today is {{ mydate|to_arrow|format_arrow('YYYY-MM-DD') }}
 <img src="{{ url_for('invenio_formatter_badges.badge', title='DOI', value='invenio.12345', ext='svg') }}"></img>
+Invenio will not be hacked! {{ content | safe | sanitize_html() }}.
```

Comment:
Should this be the other way around: ``{{content | sanitize_html | safe }}``

Reply:
That's what I thought, but if we use that order everything is escaped and the page will show the HTML markup rather than rendering it.